.github/workflows/commit.yml

```diff
@@ -1,6 +1,10 @@
 name: Commit
 
-on: push
+on:
+  push:
+  pull_request:
```

Comment:
seems to be missing some spaces here

Reply:
Nope, it is a bit confusing, but this is how you define two triggers, push and pull_request.